src/types/index.ts

```diff
@@ -63,6 +63,10 @@ export type StylesAndClasses<T> = {
 
 export type CssDimensionAbbreviation = 'h' | 'w' | 'mw' | 'mh' | 'minw' | 'minh';
 
+export type CssDimensionUnit = 'px' | 'rem' | 'em' | '%';
+
+export type CssDimensionValue = `${number}${CssDimensionUnit}`;
```

Comment:
this template string literal typescript type is causing the build to fail. tried to upgrade related ts packages but that didn't seem to resolve the issue, so I may just revert back to accepting strings like we do elsewhere.

Reply:
Looks like tsdx is dragging us down to Typescript v3.9.10. I'm playing around with the config to see if I can sort it out. I think this is also contributing to why I'm unable to symlink p-c into helios-app.

Reply:
I'm making some progress by swapping `tsdx` (abandoned before Typescript v4) with `dts-cli` (forked from tsdx to add Typescript v4 support). This essentially upgrades the whole project from Typescript v3 to Typescript v4, so I have several things to sort out. I'll post a PR when it's ready.